src/scenario/data.h

```diff
@@ -12,6 +13,7 @@
 #define MAX_INVASIONS 20
 #define MAX_DEMAND_CHANGES 20
 #define MAX_PRICE_CHANGES 20
+#define MAX_CUSTOM_VARIABLES 100
```

Comment:
Why not make this dynamic?

I'm betting map makers will find a way to hit this limit sooner rather than later...

Reply:
There are 3 options here:

1. Leave as is (i.e. not dynamic)
2. Rewrite scenario data piece to be dynamic (it currently is not).
3. Move custom variable saving out to its own piece. Probably best to move the other related functions to the new piece / file as well.

Reply:
I lean towards 1. because I don't know if map makers will use this as much. It starts touching on coding more than map making, and I don't think there are that many map makers that will actively use this.
Hence why I felt 100 is more than sufficient.

Reply:
I'd say go with option 2.

We eventually want to have dynamic number of invasions, requests etc so it's best to start preparing for it anyway.

Reply:
We both answered at the same time.

Okay, let's keep it for now.

If people start requesting more stuff in the future, we can add it then.